docs/learn.rst

```diff
@@ -25,65 +29,66 @@ To sign in to **Kolibri** and start learning follow these steps:
 	.. figure:: img/learn-page-signin.png
 		:alt: Location of "Sign in" to Kolibri links.
 
-		Location of "Sign in" to Kolibri links.
+		Location of links to "Sign in" Kolibri .
 
-Once you have logged in into Kolibri, you can see and edit your user data from the **Profile** option in the main menu (below **Learn**).
+Once you have logged in into Kolibri, you can see and edit your user data from the **Profile** option in the main sidebar menu (below **Learn**).
 
 	.. figure:: img/update-profile.png
 		:alt: View and edit your user profile.
 
 		View and edit your user profile.
 
 To sign out from Kolibri you can either:
-	* Click the user icon in the upper right corner and select **Sign out** option.
-	OR
-
-	* Select **Sign out** option in the main menu.
+	* Click the user icon in the upper right corner and select **Sign out** option, **OR**
+	* Select **Sign out** option in the main sidebar menu.
 
 
-Learn
-=====
+Learn with Kolibri
+==================
 
-Each time you login into **Kolibri**, the first thing you will see is the **Learn** page. Here you will find learning topics and materials related to what you were doing the last time you used Kolibri, or those recommended by your teachers and coaches (not visible if you are browsing as a guest). 
+Each time you login into **Kolibri**, the first thing you will see is the **Learn** page. Here you will find learning topics and materials related to what you were doing the last time you used Kolibri, or those recommended by your teachers and coaches (not visible if you are browsing as a guest).
 
 Recommended
 ^^^^^^^^^^^
 
 In the **Recommended** tab you can see various sections:
 
-* **Most popular** section which displays the most frequently used materials in your school or facility.
-* **Next steps** section which displays suggestions according to your previously visited activities. 
-* **Resume section** displays activities that you started but haven’t finished yet.
-* **Featured in...** section.  
+* **Most popular** section displays the most frequently used materials in your school or facility.
+* **Next steps** section displays suggestions according to  topics and materials you previously visited.
+* **Resume** section displays activities that you started but haven’t finished yet.
+* **Featured in...** sections display a preview of activities for each content channel you have access on Kolibri.
```

Comment:
I fail to see the _weirdness_, but not being native may explain it... :blush: 

I'd favor the second suggestion above, just remove the word `section`.

Reply:
I guess I read it like:

> Facebook PR representative released a statement.

> The Facebook PR representative released a statement.

> Facebook released a statement.

Reply:
I don't think we should make a big issue out of this, I think @indirectlylit 's suggestion seems valid, so will just change it to something that's easy to read because it doesn't make on think "hmm did I mis-read it" because of the missing "The" in from. I think this will be a benefit for translators alike.

Reply:
agreed it's minor nit-picking

Reply:
(something I do a lot of :) )